### PR TITLE
Add chain dex

### PR DESCRIPTION
**Pull request by jasonliudan**

### CODE REVIEW

File: src/multicall.ts [hunk 1]
```diff
@@ -9,29 +9,49 @@ export const multicallAddresses: { [ChainId: number]: string } = {
...
-  // [ChainId.OPT]: '',
+  [ChainId.FSN]: '0x0769fd68dFb93167989C6f7254cd0D766Fb2841F',
   [ChainId.MOR]: '0x270f2F35bED92B7A59eA5F08F6B3fd34c8D9D9b5',
-  // [ChainId.CELO]: '0x9aac9048fC8139667D6a2597B902865bfdc225d3',
-  [ChainId.HEC]: '0xdDCbf776dF3dE60163066A5ddDF2277cB445E0F3',
+  // [ChainId.EXP]: '', // TODO
+  // [ChainId.TCH]: '', // TODO
+  // [ChainId.UBQ]: '', // TODO
+  // [ChainId.MET]: '', // TODO
+  // [ChainId.DIO]: '', // TODO
```
Comment (mathiasmoeller) [inline src/multicall.ts:25]: Should this be part of this PR or do we want to resolve the TODOs later?

**Review by mathiasmoeller — CHANGES_REQUESTED** [review]
So many new tokens 💪

Regarding the images (I can't comment them directly unfortunately):
Some images like `expanse.png` should be cropped. They have a large border and they will appear pretty small when shown in the website.

File: src/multicall.ts (hunk 1 above)
Comment (jasonliudan) [reply src/multicall.ts:25]: we will add them later once they're available. so we remain them as they're.

**Comment by jasonliudan** [comment]
> So many new tokens 💪
> 
> Regarding the images (I can't comment them directly unfortunately): Some images like `expanse.png` should be cropped. They have a large border and they will appear pretty small when shown in the website.

@mathiasmoeller , I am gonna crop it to 256 x 256. will it be fine for website ?

**Comment by mathiasmoeller** [comment]
> @mathiasmoeller , I am gonna crop it to 256 x 256. will it be fine for website ?
Yes I think so, we only display them as very small icons right now :)

**Pull request merged**